src/components/BasePage/BasePage.module.css

```diff
@@ -1,10 +1,18 @@
+.pageContainer {
+  min-height: 100vh;
+  display:flex;
+  flex-direction:column;
+}
+
 .header {
   text-align: center;
   padding-top: 10px;
   padding-bottom: 10px;
 }
 
-.page {
+.pageArticle {
+  flex: 1;
+
```

Comment:
The changes to `BasePage/`and `Footer/`  are so that the footer won't cover the bottom content, and will only be visible in the viewport once at the end of the content. For reference, see this: https://stackoverflow.com/a/20352949/2877687